yb-voyager/src/srcdb/data/gather-assessment-metadata/postgresql/yb-voyager-pg-gather-assessment-metadata.sh

```diff
@@ -209,13 +213,9 @@ main() {
                     continue
                 fi
 
-                if [[ -z "$pgss_ext_schema" ]]; then
-                    print_and_log "WARN" "Skipping $script_action: pg_stat_statements extension schema is not found or not accessible."
-                    continue
-                fi
-
-                if [[ ! " ${schema_array[*]} " =~ " $pgss_ext_schema " ]]; then
-                    print_and_log "WARN" "Skipping $script_action: pg_stat_statements extension schema '$pgss_ext_schema' is not in the expected schema list (${schema_array[*]})."
+                log "INFO" "argument pgss_enabled=$pgss_enabled"
+                if [[ "$pgss_enabled" == "false" ]]; then
+                    print_and_log "WARN" "Skipping $script_action: pg_stat_statements is not enabled on source for assessment"
```

Comment:
nit: "Skipping because pgss_enabled = false". Not necessarily because "pg_stat_statements is not enabled".